protocol-configuration/mainnetAddresses.ts

```diff
@@ -66,39 +56,6 @@ export const MainnetContractsConfig = {
     address: '0x6F6580285a63f1e886548458f427f8695BA1a563',
```

Comment:
this core contract should be moved above the distribution contracts so its in the same category. Also the underlying timelocks associated with the burner contracts should be added back to the distribution category for completeness imo

Reply:
Done